.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,40 @@
+name: Cross-Browser Tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  ubuntu_tests:
+    name: Linux
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: yarn
+      - run: sudo chown -R $(whoami) $(npm config get prefix)/{lib/node_modules,bin,share}
+      - run: yarn global add gulp-cli
+      - name: Build and Test
+        run: node build-system/pr-check/cross-browser-tests-linux.js
```

Comment:
Does each OS need to have a separate node script? Would it be cleaner to pass a flag here or add environment variables?

Reply:
Good idea. There is precedent for using OS specific environment variables. I've consolidated this to a single file.